x/stakeibc/keeper/lsm_token_deposit.go

```diff
@@ -48,27 +48,52 @@ func (k Keeper) GetAllLSMTokenDeposit(ctx sdk.Context) []types.LSMTokenDeposit {
 }
 
 func (k Keeper) AddLSMTokenDeposit(ctx sdk.Context, deposit types.LSMTokenDeposit) {
-	// TODO [LSM]
-	// See if a deposit already exists for this denom
-	// If so, increment the amount
-	// otherwise, create a new deposit
+	original, found := k.GetLSMTokenDeposit(ctx, deposit.ChainId, deposit.Denom)
+	if found {
+		deposit.Amount = original.Amount.Add(deposit.Amount)
+	}
+	k.SetLSMTokenDeposit(ctx, deposit)
 }
 
-func (k Keeper) UpdateLSMTokenDepositStatus(ctx sdk.Context, deposit types.LSMTokenDeposit, status types.LSMDepositStatus) {
-	// TODO [LSM]
+func (k Keeper) UpdateLSMTokenDepositStatus(
```

Comment:
It bugs me when lines are over 80 characters but that ship has long sailed in this codebase.  I think the more descriptive names are worth it, if 140 is your line length limit, now I know.  But yes I had intentionally broken lines here because it makes it easier to read the arguments at the expense of the function opening shape is not as recognizable.

Btw I also debated getting rid of this whole function because calling it is almost the same length as setting a status on the deposit and calling to SetLSMTokenDeposit instead, but whatever, it is already done and tested.

Reply:
haha gotchya, I mostly just relax the line length on the function signature. For the rest of the code, I don't target a specific number but I probably max it around 100-110. I do find that trying to force _every_ line to <80 often makes it less readable - but obviously there's a lot of personal bias there. 

Yeah it's definitely not necessary, but since we update the status often, it's just slightly more readable to be able to do it in one line. But if you think we should remove it, I'm fine with that